synapseclient/entity.py

```diff
@@ -464,6 +464,18 @@ def __init__(self, name=None, properties=None, annotations=None, local_state=Non
         super(Project, self).__init__(concreteType=Project._synapse_entity_type, properties=properties,
                                       annotations=annotations, local_state=local_state, **kwargs)
 
+class Docker(Entity):
+    """
+    Represents a Docker in Synapse.
+
+    You cannot create a Synapse Docker image with this function, please follow instructions here (http://docs.synapse.org/articles/docker.html) to create one.
```

Comment:
Confusing about what you are creating then to have this warning...maybe this should be shown when a user creates, and an image doesn't exist (or both here and there)?